src/arrayMethodSort.js

```diff
@@ -4,9 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
-  };
+  // write code here
+  [].__proto__.sort2
+    = function(compareFunction = (a, b) => String(a) > String(b)) {
```

Comment:
let's create a separate compare function instead of creating it directly in method's parameter in order to keep our code clear.

Reply:
Also now your function returns boolean.
Our original sort method accepts functions that return's either 1, -1 or 0
Let's make it work like this as well